mautic/mautic
# Fresh reskin

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | x
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Landing page templates for Aurora, Cards, Sparse and Vibrant themes

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. Create a landing page under each of these new themes, use the editor extensively.
2. Place a favicon.ico in the assets directory of the platform, ensure the live page uses it in the browser tab
